### PR TITLE
Remove portal deprecated event usage

### PR DESCRIPTION
**Pull request by gplushi**
I fixed the deprecated event by modifying some files referencing the deprecated event and deleting files that 100% depend on removed classes. I also updated the bukkit api to 1.15. Tested and can teleport between worlds on a 1.16.5 Tuinity server.

### CODE REVIEW

**Comment by benwoo1110** [comment]
Will this cause 1.13 and 1.14 support to be dropped?

**Comment by gplushi** [comment]
Lets see

**Comment by gplushi** [comment]
org.bukkit.plugin.UnknownDependencyException: Unsupported API version 1.15 seems not, could maybe changing the API version to 1.13 when it was built on 1.15 would work...

**Comment by gplushi** [comment]
Changing API version in plugin.yml to 1.13 while staying on 1.15.2 in POM worked on a Spigot 1.13 server with just multiverse core.

**Comment by gplushi** [comment]
> Will this cause 1.13 and 1.14 support to be dropped?

Commit 5d9ea1e allows them to work

File: src/main/java/com/onarandombox/MultiverseCore/listeners/MVPortalListener.java [hunk 1]
```diff
@@ -38,13 +37,13 @@ public MVPortalListener(MultiverseCore core) {
...
     @EventHandler
-    public void entityPortalCreate(EntityCreatePortalEvent event) {
+    public void entityPortalCreate(PortalCreateEvent event) {
         if (event.isCancelled() || event.getBlocks().size() == 0) {
             return;
         }
         MultiverseWorld world = this.plugin.getMVWorldManager().getMVWorld(event.getEntity().getWorld());
         // We have to do it like this due to a bug in 1.1-R3
-        if (world != null && !world.getAllowedPortals().isPortalAllowed(event.getPortalType())) {
+        if (world != null && !world.getAllowedPortals().isPortalAllowed(PortalType.CUSTOM)) {
```
Comment (nicegamer7) [inline src/main/java/com/onarandombox/MultiverseCore/listeners/MVPortalListener.java:46]: This seems incorrect to me.

**Review by nicegamer7 — COMMENTED** [review]
I noticed one change that seemed incorrect to me. Additionally, in order to properly support 1.13, we'd have to come up with another solution to set the search radius. Removing the travel agent will break that functionality on 1.13.

File: src/main/java/com/onarandombox/MultiverseCore/listeners/MVPortalListener.java (hunk 1 above)
Comment (gplushi) [reply src/main/java/com/onarandombox/MultiverseCore/listeners/MVPortalListener.java:46]: I did that because I was getting an error there.

**Comment by gplushi** [comment]
yeah now my friend cant change worlds i dont know what im doing

**Comment by nicegamer7** [comment]
No worries, thanks for trying.

**Pull request closed without merging**